Dockerfile

```diff
@@ -1,10 +1,11 @@
 # Build Agora from source
 FROM bpfk/agora-builder:latest AS Builder
 ARG DUB_OPTIONS
+ARG AGORA_VERSION="HEAD"
```

Comment:
this is deliberately not an ENV as I would not like to preserve this environment variable after the docker build finishes